lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
I think in this case we'd want to do a "collect like terms" where for example:

3 * 10^6 * 5 * 10^2 => 3 * 5 * (10^6 * 10^2) and then we combine exponents

this already happens with symbols in cases like

`y * x^6 * z * x^2 => y * z * (x^6 * x^2)`

so I'd suggest following that kind of pattern :)